src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/AdvSimd.PlatformNotSupported.cs

```diff
@@ -46,6 +46,18 @@ public new abstract class Arm64 : ArmBase.Arm64
             /// </summary>
             public static Vector128<double> Add(Vector128<double> left, Vector128<double> right) { throw new PlatformNotSupportedException(); }
 
+            // /// <summary>
+            // /// float64x1_t vbsl_f64 (uint64x1_t a, float64x1_t b, float64x1_t c)
+            // ///   A64: BSL Vd, Vn, Vm
+            // /// </summary>
+            // public static Vector64<double> BitwiseSelect(Vector64<ulong> select, Vector64<double> left, Vector64<double> right) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// float64x2_t vbslq_f64 (uint64x2_t a, float64x2_t b, float64x2_t c)
+            ///   A64: BSL Vd, Vn, Vm
+            /// </summary>
+            public static Vector128<double> BitwiseSelect(Vector128<ulong> select, Vector128<double> left, Vector128<double> right) { throw new PlatformNotSupportedException(); }
```

Comment:
For x86, we had determined that the mask/select inputs should be the same type as the operands (so `double/float` rather than `ulong/uint` or `long/int`).

This was done because the output of other common operations that are used for such masks (such as `And`, `AndNot`, `Vector Compare`, etc) all return the same type as was input.

Reply:
Same goes for the `short/long/int` overloads, which should take `short/long/int` as the mask

Reply:
Done